README.md

```diff
@@ -46,39 +46,58 @@ php-fpm with PHP 7.2
 
 1. Get [Docker for OSX](https://download.docker.com/mac/stable/Docker.dmg) and install it.
     - Do not forget to tune up the allocated Memory and CPUs. `Docker` > `Preferences` > `Advanced`
-1. Create a `repositories` folder
-1. Clone all the wanted repositories in there.
-    - [https://github.com/vanilla/vanilla](vanilla/vanilla)
-        - It is recommended to use "vanilla_dev" as the database name since some services are configured to use that database.
-    - [https://github.com/vanilla/vanilla-docker](vanilla/vanilla-docker)
-    - [https://github.com/vanilla/addons](vanilla/addons)
-    - ...
+1. Get [Composer](https://getcomposer.org/) and install it.
+1. Create a directory for your project. In this example, we'll use `my-vanilla-project`, but you can use any name.
+1. Move into your project directory.
+1. Clone the following repos into your project directory:
+    1. [https://github.com/vanilla/vanilla](vanilla/vanilla)
```

Comment:
All of your links are backwards:

```md
_Incorrect_
[https://github.com/vanilla/vanilla](vanilla/vanilla)
_Correct_
[vanilla/vanilla](https://github.com/vanilla/vanilla)
```

Reply:
Good catch. These are actually the existing links, my only change was to make it an ordered list. I'll make the fix